npm/angular/cypress/plugins/webpack.config.ts

```diff
@@ -1,5 +1,5 @@
-import * as webpack from 'webpack'
 import * as path from 'path'
+import { AngularWebpackPlugin } from '@ngtools/webpack'
```

Comment:
It sounds like this will be a blocker for merging, right? What's the urgency like for Angular 12.x support?

Reply:
Yeah I am actively working on this locally. Once I find a solution locally, I will attempt to add these changes here. I will say that when we release Angular CT we need to support the most recent version of Angular which is actually 13. There is a large number of Angular users who use the most recent versions. Currently we are unable to upgrade this app here to 13 though as Angular requires a minimum Node version of 16.10 and we are using 16.5. My plan was to get to 12 and then create a separate PR to get to 13 because of that

Reply:
Okay, great. Do you mind setting this to draft until it's read for a re-review?